### ISSUE TITLE
Schema update required?

### ISSUE THREAD

**Issue by baoshan** [linuxserver/docker-musicbrainz#7]
```
Wed Jun  1 14:37:36 2016 : Pre-checking /data/import/20160528-001701/mbdump-cdstubs.tar.bz2

bunzip2: I/O or other error, bailing out.  Possible reason follows.
bunzip2: Broken pipe
    Input file = (stdin), output file = (stdout)
Wed Jun  1 14:37:36 2016 : Schema sequence mismatch - codebase is 22, /data/import/20160528-001701/mbdump-cdstubs.tar.bz2 is 23

Failed to import dataset.
```

Does this mean an update should be performed? Thank you.

**Comment by baoshan** [comment]
Is #6 the correct way to go?

**Comment by baoshan** [comment]
Solved by #8.
